.github/workflows/release.yml

```diff
@@ -84,7 +84,8 @@ jobs:
           private_key: ${{ secrets.OBS_AUTOMATION_APP_PEM }}
           permissions: >-
             {
-              "contents": "write"
+              "contents": "write",
+              "pull-requests: "write"
```

Comment:
<img width="646" alt="image" src="https://github.com/user-attachments/assets/8cbc5e8f-1405-489d-bbf2-b5871c488ca7">

see https://docs.github.com/en/rest/apps/apps?apiVersion=2022-11-28#create-an-installation-access-token-for-an-app under the `permissions` collapsed details

```suggestion
              "pull_requests: "write"
```

Reply:
Woops, thanks!

Reply:
no worries, it's a bit of pain in the `heck`, I faced it that issue a few times in the past 😮‍💨